include/ur_api.h

```diff
@@ -7542,13 +7542,14 @@ typedef enum ur_exp_image_copy_flag_t {
     UR_EXP_IMAGE_COPY_FLAG_HOST_TO_DEVICE = UR_BIT(0),   ///< Host to device
     UR_EXP_IMAGE_COPY_FLAG_DEVICE_TO_HOST = UR_BIT(1),   ///< Device to host
     UR_EXP_IMAGE_COPY_FLAG_DEVICE_TO_DEVICE = UR_BIT(2), ///< Device to device
+    UR_EXP_IMAGE_COPY_FLAG_HOST_TO_HOST = UR_BIT(3),     ///< Host to host
     /// @cond
     UR_EXP_IMAGE_COPY_FLAG_FORCE_UINT32 = 0x7fffffff
     /// @endcond
 
 } ur_exp_image_copy_flag_t;
 /// @brief Bit Mask for validating ur_exp_image_copy_flags_t
```

Comment:
Ignore if I am out of the loop, but is it expected that this mask changes from `0xfffffff8` to `0xfffffff0`?

Reply:
I did not manually touch any of the ur_* files. Regenerating the files changed it to this. after adding an entry to this enum. Chatting to Georgi he says that this is expected behaviour.

Reply:
> Ignore if I am out of the loop, but is it expected that this mask changes from `0xfffffff8` to `0xfffffff0`?

Yes, the mask is used in validation of the bit set so adding a new enumeration should update the mask to capture the additional valid input.